docs/semgrep-ci/semgrep-ci-with-app.md

```diff
@@ -0,0 +1,332 @@
+* Scan many repositories and manage their findings in bulk.
+* Triage (ignore) false-positive findings from low-performing rules.
+* Fork existing rules to create custom rules and add them to Semgrep App for scanning.
+
+This guide explains how to connect your repository to Semgrep App to scan continuously.
+
+:::note
+* This guide's configuration and feature support are specific to Semgrep-App-connected CI jobs. Refer to [Running Semgrep in CI without Semgrep App](../manual-ci-setup) for stand-alone CI jobs.
+* Semgrep 0.98.0 introduced changes to how certain CI providers fetch environment variables. Refer to the appendix at the end of this document for more information.
+:::
+
+The following video walks you through setting Semgrep in your CI through Semgrep App.
+
+<iframe class="yt_embed" width="100%" height="432px" src="https://www.youtube.com/embed/ukIUM3j0gZY" frameborder="0" allowfullscreen></iframe>
+
+
+## Feature support
+
+Support for certain features of Semgrep App depend on your CI provider or source code management tool (SCM). The following table breaks down the features and their availability:
+
+| Feature | GitHub | GitLab | BitBucket | CI Provider support |
+| ------- | -------- | ------- | -------- | ---------------- |
+| **Diff-aware scanning** | ✅ Yes | ✅ Yes | ✅ Yes  | ✅ Available (may need additional set up) | 
+| **Hyperlinks** | ✅ Yes | ✅ Yes | ✅ Yes  |  ✅ Available (may need additional set up) |
+| **SCM security dashboard** |  ✅ GitHub Advanced Security Dashboard |  ✅ GitLab SAST Dashboard | ❌ No | ❗ Only GitHub Actions and GitLab CI/CD |
+| **PR or MR comments** |  ✅ Yes | ✅ Yes | ❌ No | ✅ CI provider agnostic; feature support is dependent on SCM |
+*Table 1.* List of features and supported SCMs and CI providers.
+
+<dl>
+    <dt>Diff-aware scanning</dt>
+    <dd>Semgrep App can scan changes in files when running on a pull or merge request (PR or MR). This keeps the scan fast and reduces finding duplication.</dd>
+    <dt>Receiving results (findings) as PR or MR comments</dt>
+    <dd>This feature enables you to receive <a href="/docs/semgrep-app/notifications/#enabling-github-pull-request-comments">PR or MR comments</a> from Semgrep App on the lines of code that generated a finding.</dd>
+    <dt>Hyperlinks to code</dt>
+    <dd>Semgrep App collects findings in a Findings page. In this page, you can click on a finding to view the lines of code in your repository that generated the finding.</dd>
+    <dt>SCM security dashboard</dt>
+    <dd>Send Semgrep findings to your SCM's security dashboard.</dd>
+</dl>
+
+## Setting up the CI job and Semgrep App connection
+
+![Steps to run Semgrep in CI without Semgrep App](/img/semgrep-ci-overview-app.png "Steps to integrate Semgrep in CI with Semgrep App")
+
+*Figure 1.* Steps to run Semgrep in CI with Semgrep App.
+
+Refer to the succeeding sections for guidance specific to your CI provider.
+
+### In-App Providers (such as GitHub Actions, GitLab CI/CD, Jenkins)
```

Comment:
Trying to keep the perspective of a newer reader, I am not sure what “In-App Providers” means.

```suggestion
### In-App providers (such as GitHub Actions, GitLab CI/CD, Jenkins)
```

Reply:
Changing to "CI providers listed within Semgrep App" - wordy but clearer.